downloads/swan-lake-release-notes/swan-lake-2201.7.0/RELEASE_NOTE.md

```diff
@@ -61,6 +84,22 @@ public function main() {
 ```
 
 ### Improvements
+- Made the type node in module-level constant declarations optional. Both of the following cases are valid.
```

Comment:
We usually add titles for improvements.  See https://ballerina.io/downloads/swan-lake-release-notes/swan-lake-2201.3.0#improvements.

Reply:
Added